src/atomate2/classical_md/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""Module for classical md workflows."""
```

Comment:
To me this folder should not be called `classical_md` but instead `openff`. And the `openmm` subdirectory should be its own folder in the base dir. E.g., `atomate2/openmm`